### DISCUSSION TITLE
interaktif plot tdk keluar setelah diknit

### DISCUSSION THREAD

**Discussion by manuelatito** [teamalgoritma/community#882] [Q&A] [answered]
Dear Team Algoritma,

Saya ingin bertanya.. Interactive plot sy tdk keluar sepenuhnya setelah di knit, hasilnya seperti di bawah ini.
Mohon bantuannya, apakah ada yg salah/terlewat dalam proses knit sehingga tdk bisa mengeluarkan plot sy?
Terima kasih

![image](https://github.com/teamalgoritma/community/assets/164650405/f4247cf0-584a-47bb-a014-676142635cd5)

![image](https://github.com/teamalgoritma/community/assets/164650405/e77538b9-a9d8-49ec-9490-7e5b6df618ea)

![image](https://github.com/teamalgoritma/community/assets/164650405/5645f5dd-316a-4e8e-9d0e-ef0440a411fd)

**Comment by ramdhinafinita** [comment, marked as answer]
Dear @manuelatito 

Untuk penulisan kode di flexdashboard.rmd bisa di cek kembali ya bu, yang dimasukkan kedalam plot hanya kode untuk plot. 

1. Bisa di cek dan disamakan untuk format seperti yang dikelas. Perhatikan ### nya juga karena seharusnya untuk output dataframe/object jangan sampai dipanggil kembali agar tidak menghasilkan output dashboard.

2. Bisa cek kembali untuk kode ggplotlynya apakah sudah benar atau belum. 

Jika masih terkendala, bisa kirimkan rmd dan dataset ke email [product@algoritmadata.id](https://mail.google.com/mail/?view=cm&fs=1&to=product%40algoritmadata.id&authuser=0)

Terimakasih, semoga berhasil

**Reply by manuelatito** [reply]
Dear kak @ramdhinafinita,

Terima kasih banyak kak Fini, sy sudah berhasil atas bimbingan kk.. :)
